tests/python/PyImportSortBearTest.py

```diff
@@ -21,3 +21,10 @@
      'import abc\nimport xyz\n'),
     settings={'known_standard_library_imports': 'xyz',
               'known_first_party_imports': 'abc'})
+
+PyImportSortBearIgnoredConfigsTest = verify_local_bear(
```

Comment:
This TestCase has the same name as the on in line 16. Therefore the other one is shadowed (never executed). This is why you loose the coverage for the `known_standard_library_imports` if you remove it from the new Testcase.